spec/tree-view-spec.coffee

```diff
@@ -2588,6 +2588,43 @@ describe "TreeView", ->
           omicronDir = $(treeView.roots[0].entries).find(".directory:contains(omicron):first span")[0]
           expect(omicronDir.title).toEqual("omicron")
 
+      describe "when a file is created within a directory with another squashed directory", ->
+        it "un-squashes the directories", ->
+          jasmine.attachToDOM(workspaceElement)
+          piDir = $(treeView.roots[0].entries).find(".directory:contains(omicron#{path.sep}pi):first")[0]
+          expect(piDir).not.toBeNull()
+          omicronPath = piDir.getPath().replace "/pi", ""
```

Comment:
This part doesn't seem Windows-friendly.  Can't you do the same logic for `piDir` and change it something like `omicronPath = $(treeView.roots[0].entries).find(".directory:contains(omicron):first")[0].getPath()`?